plugins/modem-manager/fu-mm-device.c

```diff
@@ -165,6 +168,9 @@ validate_firmware_update_method(MMModemFirmwareUpdateMethod methods, GError **er
 #endif /* MM_CHECK_VERSION(1,17,1) */
 #if MM_CHECK_VERSION(1, 17, 2)
 		MM_MODEM_FIRMWARE_UPDATE_METHOD_FIREHOSE,
+#endif
+#if MM_CHECK_VERSION(1, 19, 0)
```

Comment:
Please use 1.19.1, I just added that version bump to flag the existence of the new SAHARA enum value